pkgs/applications/radio/sdr-j-fm/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  wrapQtAppsHook,
+  pkg-config,
+  qtbase,
+  qwt,
+  fftwFloat,
+  libsamplerate,
+  portaudio,
+  libusb1,
+  libsndfile,
+  airspy,
+  rtl-sdr,
+  sdrplay,
```

Comment:
This package is `unfree`. One could consider to make it optional.

Reply:
> This package is `unfree`. One could consider to make it optional. (I have no strong opinion on this)

Thanks for noticing this :) Apparently, the 3 dependencies `airspy`, `rtl-sdr` and `sdrplay` weren't really needed (I don't remember why I added them in the first place). So they are removed now.